tool/src/org/antlr/v4/codegen/target/CSharpTarget.java

```diff
@@ -173,7 +173,7 @@ private void reportError(STMessage msg) {
 
 	@Override
 	public boolean isATNSerializedAsInts() {
-		return false;
+		return true;
```

Comment:
I guess this method should be completely removed since abstract method has the same implementation.

Reply:
I will take care of this once I get rid of more bring atn stuff.